src/config/ConfigParser.ts

```diff
@@ -116,6 +116,38 @@ export function configParse5(slice: Slice | null | undefined) {
     throw new Error('Invalid config');
 }
 
+export function configParse6(slice: Slice | null | undefined) {
+    if (!slice) {
+        // no param in mainnet for now, so throwing will cause crash of parseFullConfig()
+        return null
+    }
+    const mintNewPrice = slice.loadCoins();
+    const mintAddPrice = slice.loadCoins();
+    return {
+        mintNewPrice,
+        mintAddPrice
+    };
+}
+
+export function configParse7(slice: Slice | null | undefined) {
```

Comment:
```
_ to_mint:ExtraCurrencyCollection = ConfigParam 7;
```

`toMint` is absent in return type. Even though it's the only parameter at the moment, schema might be extended with extra parameters in the future, and there'll be other fields beside `toMint`.

A separate `loadExtraCurrencyCollection` function should be used. Then if someone prefers to (precariously) parse parameter without the extra `{ toMint: ... }`, they might prefer to call it directly.